.buildkite/publish-react-native-ios-artifacts.sh

```diff
@@ -8,4 +8,12 @@ install_gems
 echo "--- :cocoapods: Setting up Pods"
 install_cocoapods
 
+echo "--- 🚧 Install xcbeautify formatter while not on the VM image"
+# The env vars should make Homebrew run faster
+HOMEBREW_NO_AUTO_UPDATE=1 HOMEBREW_NO_INSTALL_CLEANUP=1 brew install xcbeautify
+
+echo "--- :xcode: Build XCFramework"
 ./build.sh
+
+# Restore initial the location in the filesystem location
+cd ..
```

Comment:
This command should have no effect right? Because it just changes working directory for nothing (no code after this line)?

Reply:
Yes it's "useless". I remember having some issue with artifacts upload and tried this, but it didn't help.